flow-plugins/vaadin-gradle-plugin/src/functionalTest/kotlin/com/vaadin/gradle/MiscSingleModuleTest.kt

```diff
@@ -0,0 +1,436 @@
+package com.vaadin.gradle
+
+import com.vaadin.flow.server.frontend.FrontendUtils
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.Test
+import java.io.File
+import kotlin.test.expect
+
+class MiscSingleModuleTest : AbstractGradleTest() {
+    /**
+     * Tests https://github.com/vaadin/vaadin-gradle-plugin/issues/26
+     */
+    @Test
+    fun testVaadin8Vaadin14MPRProject() {
+        testProject.buildFile.writeText(
+            """
+            plugins {
+                id "com.devsoap.plugin.vaadin" version "1.4.1"
+                id 'com.vaadin'
+            }
+            repositories {
+                mavenCentral()
+                jcenter()
+                maven { url = 'https://maven.vaadin.com/vaadin-prereleases' }
+            }
+            // test that we can configure both plugins
+            vaadin {
+                version = "8.9.4"
+            }
+            vaadin14 {
```

Comment:
```suggestion
            vaadinPlatform {
```
The custom target should not be named 14 as this plugin version doesn't work with that. `Platform` is better.